### PR TITLE
CartOrder and ReOrder mutations

### PR DESCRIPTION
**Pull request by DenisMela**
Tested working

The GraphQL mutation https://github.com/cheapreats/CheaprEats-GraphQL-API/pull/436

**Checklist:**
- [ ] Changed package version (must be higher than **master**)
- [ ] Updated README (if applicable)

### CODE REVIEW

File: src/app/controllers/OrderController.ts [hunk 1]
```diff
@@ -42,6 +42,15 @@ export interface CreateOrderInput {
   scheduled_pickup: string;
   order_type?: OrderType; // default TAKE_OUT
 }
+export interface CreateOrderFromCartInput {
+  payment_method: string;
```
Comment (ralph-dev) [inline src/app/controllers/OrderController.ts:46]: Is this a string or enum?

File: src/app/controllers/OrderController.ts [hunk 2]
```diff
@@ -42,6 +42,15 @@ export interface CreateOrderInput {
...
+  cart_id: string;
+  scheduled_pickup: string;
```
Comment (ralph-dev) [inline src/app/controllers/OrderController.ts:48]: Is there a typescript Date type?

**Review by ralph-dev — APPROVED** [review]
Lgtm

**Comment by ralph-dev** [comment]
https://www.typescriptlang.org/docs/handbook/2/basic-types.html#explicit-types

Not sure if date or string makes sense here depends on backend

**Comment by ralph-dev** [comment]
@DenisMela Make this a PR if you want a real review

**Review by ralph-dev — COMMENTED** [review]
LGTM

File: src/app/controllers/OrderController.ts [hunk 3]
```diff
@@ -42,6 +37,15 @@ export interface CreateOrderInput {
   scheduled_pickup: string;
   order_type?: OrderType; // default TAKE_OUT
 }
+export interface CreateOrderFromCartInput {
+  payment_method: OrderPaymentMethod;
+  cart_id: string;
+  scheduled_pickup: Date;
```
Comment (ralph-dev) [inline src/app/controllers/OrderController.ts:43]: @oopsidoodles Did you know about this type 👍 
@junzhengca This is good right?

**Review by ralph-dev — APPROVED** [review]
LGTM, will wait for others

**Review by junzhengca — COMMENTED** [review]
LGTM, Approval

**Review by junzhengca — APPROVED** [review]

**Pull request merged**